core/sparqlbuilder/src/main/java/org/eclipse/rdf4j/sparqlbuilder/core/Variable.java

```diff
@@ -21,15 +21,19 @@
  */
 public class Variable implements Projectable, RdfSubject, RdfPredicate, RdfObject, Operand, Orderable, Groupable,
 		GraphName, Assignable {
-	private String alias;
+	private String varName;
 
-	Variable(String varName) {
-		this.alias = varName;
+	public Variable(String varName) {
```

Comment:
I have no _very_ strong feelings on this, but if you can leave it package-private without having to jump through too many hoops, then I think I'd prefer sticking with that.

Reply:
Ok, I'll make it package-private (thanks for making me look that up btw :-)) and we'll go through `SparqlBuilder.var()` in `rdf4j-spring`.

Reply:
done